### ISSUE TITLE
order.success.additional.info container rendered twice on checkout success page

### ISSUE THREAD

**Issue by roma84** [magento/magento2#4999]
## Steps to reproduce
1. Install Magento from `develop` branch.
2. Enable and configure Paypal Express and billing agreements.
3. Sign in.
4. Add Product to cart.
5. Go to checkout.
6. Select Paypal.
7. Check Billing Agreement checkbox.
8. You will be redirected to success page after payment completed.
## Expected result
1. Billing agreement number rendered in page content
## Actual result
1. Billing agreement number rendered twice: in page content and page bottom.
2. Screenshot: 
   ![success-page-bug](https://cloud.githubusercontent.com/assets/2668851/16007430/ff605ef8-317a-11e6-95ce-43cc44f3974b.png)
## Possible solution

in `app/code/Magento/Checkout/view/frontend/layout/checkout_onepage_success.xml` move `order.success.additional.info` inside `checkout.success` block: 

``` xml
<referenceContainer name="content">
    <block class="Magento\Checkout\Block\Onepage\Success" name="checkout.success" template="success.phtml" cacheable="false">
        <container name="order.success.additional.info" label="Order Success Additional Info"/>
    </block>
        ...
</referenceContainer>
```

**Comment by ishakhsuvarov** [comment]
@roma84 Thank you for reporting this issue. We have created an internal ticket `MAGETWO-54380` to fix the problem.

**Comment by qbo-tech** [comment]
Confirmed issue on 2.1.3....

**Comment by magento-engcom-team** [comment]
@roma84, thank you for your report.
The issue is already fixed in 2.2.0
